nmproxy/router/iptables_linux.go

```diff
@@ -248,12 +248,22 @@ func (i *iptablesManager) AddIngressRoutingRule(server, extPeerKey, extPeerAddr
 	if err != nil {
 		logger.Log(1, fmt.Sprintf("failed to add rule: %v, Err: %v ", ruleSpec, err.Error()))
 	}
+	ruleSpecForward := []string{"-i", "netmaker", "-j", "ACCEPT"}
+	err = iptablesClient.Insert(defaultIpTable, netmakerFilterChain, 1, ruleSpecForward...)
```

Comment:
not the right place to add this rule
add it under `InsertEgressRoutingRules` and `InsertIngressRoutingRules`  funcs

Reply:
deleted changes.  will add to in a separate PR.